### PR TITLE
Creatable safe test

### PR DESCRIPTION
**Pull request by asnahassan**
- [x] https://github.com/intergiro/model-acquirer/pull/357 
Created a test 'Creatable.Safe types' for Authorization and Verification

### CODE REVIEW

**Review by nilssonemma — APPROVED** [review]

File: Verification/Creatable/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	})
+	it("safe is not (wrong spelling Country)", () => {
+		const safe = {
+			number: "string",
+			items: [{ price: 30 }, { price: 20 }],
+			currency: "EUR",
+			card: {
+				iin: "411111",
+				last4: "5455",
+				scheme: "viDDsa",
```
Comment (Magnus93) [inline Verification/Creatable/index.spec.ts:66]: Incorrect spelling on `scheme` as well, so this test does not prove more then the previous test.
Fix `scheme` spelling on this test

**Review by Magnus93 — CHANGES_REQUESTED** [review]
You seem to have 2 identical test files, testing the same things. Remove one of them.

File: Verification/Creatable/Safe.spec.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	})
+	it("safe is not (input wrong spelling for country)", () => {
+		const safe = {
+			number: "string",
+			items: [{ price: 30 }, { price: 20 }],
+			currency: "EUR",
+			card: {
+				iin: "411111",
+				last4: "5455",
+				scheme: "viDDsa",
```
Comment (Magnus93) [inline Verification/Creatable/Safe.spec.ts:66]: The spelling of scheme is wrong here just like in the previous test. SO this test does not prove anything more than the previous test.

**Fix `scheme` spelling.**

**Review by Magnus93 — CHANGES_REQUESTED** [review]

File: Verification/Creatable/Safe.spec.ts (hunk 2 above)
Comment (asnahassan) [reply Verification/Creatable/Safe.spec.ts]: Fixed

**Review by Magnus93 — APPROVED** [review]

**Pull request merged**